<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileDialog</class>
 <widget class="QDialog" name="ProfileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="labelName">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Futura</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="birthdateLabel">
       <property name="text">
        <string>Birthdate</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateEdit" name="birthdateDateEdit">
       <property name="minimumSize">
        <size>
         <width>423</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="birthCityLabel">
       <property name="text">
        <string>Birth City</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="birthCityLineEdit">
       <property name="minimumSize">
        <size>
         <width>423</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="birthCountryLabel">
       <property name="text">
        <string>Nationality</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="birthCountryLineEdit">
       <property name="minimumSize">
        <size>
         <width>423</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="ethnicityLabel">
       <property name="text">
        <string>Ethnicity</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="ethnicityLineEdit">
       <property name="minimumSize">
        <size>
         <width>423</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="eyeColorLabel">
       <property name="text">
        <string>Eye Color</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="eyeColorLineEdit">
       <property name="minimumSize">
        <size>
         <width>423</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="hairColorLabel">
       <property name="text">
        <string>Hair Color</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="hairColorLineEdit">
       <property name="minimumSize">
        <size>
         <width>423</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="heightLabel">
       <property name="text">
        <string>Height</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="heightLineEdit">
       <property name="minimumSize">
        <size>
         <width>423</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="measurementsLabel">
       <property name="text">
        <string>Measurements</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="measurementsLineEdit">
       <property name="minimumSize">
        <size>
         <width>423</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="weightLabel">
       <property name="text">
        <string>Weight</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="weightLineEdit">
       <property name="minimumSize">
        <size>
         <width>423</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="piercingsLabel">
       <property name="text">
        <string>Piercings</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QTextEdit" name="piercingsEdit">
       <property name="minimumSize">
        <size>
         <width>423</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="tattoosLabel">
       <property name="text">
        <string>Tattoos</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QTextEdit" name="tattoosEdit">
       <property name="minimumSize">
        <size>
         <width>423</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="aliasesLabel">
       <property name="text">
        <string>Aliases</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QTextEdit" name="aliasesEdit">
       <property name="minimumSize">
        <size>
         <width>423</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="headshotLocationLabel">
       <property name="text">
        <string>Headshot Location</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLineEdit" name="headshotLocationLineEdit">
       <property name="minimumSize">
        <size>
         <width>423</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="profilePhoto">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>200</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="tryAgainButton">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Find Photo</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
